### PR TITLE
Setup a i18n scanner command - Closes #769

### LINKED ISSUE #769: Setup a i18n scanner command
### Expected behaviour
There should be a command to run that generates the `*.json` translation source file, based on occurances of `this.props.t(`, `props.t(`, `t(`, and `i18next.t(`
e. g. https://github.com/i18next/i18next-scanner/wiki/Integration-Guide#react
### Actual behaviour
There's none

### PR DESCRIPTION
**Pull request by slaweet**
Closes #769

### CODE REVIEW

File: src/i18n-scanner.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const fs = require('fs');
+const glob = require('glob');
+const Parser = require('i18next-scanner').Parser;
+
+const translationFunctionNames = ['i18next.t', 'props.t', 'this.props.t', 't'];
+const outputFilePath = './src/locales/en/common.json';
```
Comment (reyraa) [inline src/i18n-scanner.js]: Are we supposed to update the json file of other languages manually? I don't mean the values, but the keys.

**Comment by reyraa** [comment]
How it keeps track of the changed/removed translations? say, If for any reason we change a title's casing (e.g from transaction to Transaction), will we have to search git history for the missing translation?

File: src/locales/en/common.json [hunk 2]
```diff
@@ -1,58 +1,57 @@
...
+  "confirmations": "confirmations",
   "my votes": "my votes",
-  "Search": "Search",
-  "Name": "Name",
-  "Lisk Address": "Lisk Address",
-  "Uptime": "Uptime",
-  "No delegates found": "No delegates found"
-}
+  "your second passphrase will be required for all transactions sent from this account": "your second passphrase will be required for all transactions sent from this account"
+}
```
Comment (reyraa) [inline src/locales/en/common.json]: I guess we need an ending line here

File: src/i18n-scanner.js (hunk 1 above)
Comment (slaweet) [reply src/i18n-scanner.js]: Other translation files are supposed to be changed by the localization platform set up in https://github.com/LiskHQ/lisk-nano/issues/736

In the end, I'd prefer to .gitignore the translation files and fetching them from the localization platform on a build.

**Comment by reyraa** [comment]
So the remaining points to be cleared are:
 - How does it handle different translations in different contexts
 - How does it handle plural/singular translations

**Comment by slaweet** [comment]
context and plurals is something that `i18n-scanner` can take care of based on `context` and `count` options, e.g. `props.t('{{count}} confirmations', {count: 2, context: 'tooltip'})` will by default create keys `{{count}} confirmations_tooltip`, `{{count}} confirmations_tooltip_plural`.

But I broke it by using `customHandler` to get the values prepopulated:
https://github.com/LiskHQ/lisk-nano/blob/17b192ed9334d831aa4937a77aa4ca40f28fedf7/src/i18n-scanner.js#L10-L12

I don't know how to get values prepopulated at the same time as context and plurals

**Comment by slaweet** [comment]
I gave it another try and fixed the plurals and context

**Pull request merged**